README.md

```diff
@@ -76,13 +71,16 @@ build-scxa-analytics-suggestions.sh
 ```
 
 ## Load data
-This module loads data from a condensed SDRF in an SCXA experiment to the scxa-analytics-v? collection in Solr. These routines expect the collection to be created already, and work as an update to the content of the collection.
+This module loads data from a condensed SDRF in an SCXA experiment to the
+`scxa-analytics-v6` collection in Solr. These routines expect the collection to
```

Comment:
Might be good to add a working directory provision if needed (how to set a working directory), to avoid loads of different experiments to step on each other.